File: java/com/epicdragonworld/gameserver/model/holders/SpawnHolder.java
package com.epicdragonworld.gameserver.model.holders;

import com.epicdragonworld.gameserver.model.Location;

/**
 * @author Pantelis Andrianakis
 */
public class SpawnHolder
{
	private final Location _location;
	private final int _respawnTime;
	
	public SpawnHolder(Location location, int respawnTime)
	{
		_location = location;
		_respawnTime = respawnTime;
	}
	
	public Location getLocation()
	{
		return _location;
	}
	
	public int getRespawnTime()
	{
		return _respawnTime;
	}
}

File: EpicDragonWorldServer/Holders/SpawnHolder.cs
/**
 * Author: Pantelis Andrianakis
 * Date: November 28th 2019
 */
public class SpawnHolder
{
    private readonly LocationHolder location;
    private readonly int respawnDelay;

    public SpawnHolder(LocationHolder location, int respawnDelay)
    {
        this.location = location;
        this.respawnDelay = respawnDelay;
    }

    public LocationHolder GetLocation()
    {
        return location;
    }

    public int GetRespawnDelay()
    {
        return respawnDelay;
    }
}
